### DISCUSSION TITLE
PyAnsys install quesion.

### DISCUSSION THREAD

**Discussion by whplace** [ansys/pyansys#435] [Q&A] [answered]
Hi,
I'm not really sure,
but ansys-mapdl-reader 0.53.0 seems to be unable to be installed in python3.8 ？

**Comment by RobPasMue** [comment, marked as answer]
Hi @whplace - in case you want to use ansys-mapdl-reader on Python 3.8, you need to build the package from scratch. Support for Python 3.8 was dropped in most of our PyAnsys libraries starting on October and following [NEP-0029](https://numpy.org/neps/nep-0029-deprecation_policy.html)
